src-cpp/CMakeLists.txt

```diff
@@ -17,3 +17,17 @@ target_link_libraries(rdkafka++ PUBLIC rdkafka)
 
 # Support '#include <rdkafcpp.h>'
 target_include_directories(rdkafka++ PUBLIC "$<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}>")
+
+install(
```

Comment:
No. It's a separate stuff. Installed `RdKafkaConfig.cmake` play the same role as `*.pc` files (but more powerful).